specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/WebApps.json

```diff
@@ -22122,6 +22481,48 @@
         }
       }
     },
+    "RemotePrivateEndpointConnection": {
+      "description": "A remote private endpoint connection",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "type": "string",
+          "readOnly": true
+        },
+        "privateEndpoint": {
+          "$ref": "#/definitions/ArmIdWrapper",
+          "description": "PrivateEndpoint of a remote private endpoint connection"
+        },
+        "privateLinkServiceConnectionState": {
+          "$ref": "#/definitions/PrivateLinkConnectionState"
+        }
+      }
+    },
+    "RemotePrivateEndpointConnectionWrapper": {
+      "description": "Wrapper for a remote private endpoint connection",
+      "required": [
+        "properties"
+      ],
+      "type": "object",
+      "properties": {
+        "id": {
```

Comment:
Rather than duplicate common properties (id, type, name, properties{}) in multiple definitions, it's cleaner to use a common definition with an allOf reference.  This is demonstrated on line 21872.

Reply:
Removed this as it actually wasnt being used :)